scripts/reload-as2-container.sh

```diff
@@ -4,4 +4,9 @@
 # Usage: ./scripts/reload-as2-container.sh
 # Note: This is not required for partnership.xml changes
 
+git reset --hard origin/main
+git pull
```

Comment:
this should be either git fetch, then reset - or just git pull. I prefer the fetch/reset though.